app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt

```diff
@@ -3,11 +3,13 @@ package org.simple.clinic.teleconsultlog.medicinefrequency
 import com.spotify.mobius.Next
 import com.spotify.mobius.Update
 import org.simple.clinic.mobius.dispatch
+import org.simple.clinic.mobius.next
 
 class MedicineFrequencyUpdate : Update<MedicineFrequencyModel, MedicineFrequencyEvent, MedicineFrequencyEffect> {
   override fun update(model: MedicineFrequencyModel, event: MedicineFrequencyEvent): Next<MedicineFrequencyModel, MedicineFrequencyEffect> {
     return when (event) {
-      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(model.medicineFrequency))
+      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(event.medicineFrequency))
```

Comment:
We no longer have to pass in the `MedicineFrequency` using the event because we update model with the latest `MedicineFrequency` every time user selects a new frequency.

Reply:
> Over here we are supposed to pass the event MedicineFrequency instead of the model because we are not updating the model MedicineFrequency yet.

Both of the comments say different things for the same event. However, updating it to receive from the model because now we observe the `radioButton` click in `MedicineFrequencyChanged` event.

Reply:
The reason for it was the order of commits. In an older commit, you have used model to get the `MedicineFrequency` while it was not being updated. You can go ahead with this comment change and ignore another comment because we squashed the PR.